docs/knowledge_base/api-keys.md

```diff
@@ -6,9 +6,60 @@ hidden: false
 
 API keys provide you with the ability to authenticate with the Phylum API without requiring user credentials. API keys are well suited for CI/CD environments where you may not want to disclose your account information.
 
-The `offline_access` parameter in the `settings.yaml` file contains the API token. The following command can be used to retrieve your token value:  
+## Generate an API key
+
+API keys can be generated in the [Phylum web UI][ui-tokens] by clicking on user menu in the top right and clicking on "API Access Tokens"
+
+[ui-tokens]: https://app.staging.phylum.io/tokens/list
+
+![user menu](../../assets/user-menu.png)
+
+You will be prompted to provide a name for the new key and then will be given a chance to copy the newly generated API key.
+
+**Important Note:** For security, there is no way to retrieve the value of a API key after initial generation. It is up to the user to securely store the key for later use.
```

Comment:
```suggestion
> ⚠️ For security, there is no way to retrieve the value of an API key after initial generation. It is up to the user to securely store the key for later use.
```

Not a mandatory change, but this might highlight this important note more in the docs.